### PR TITLE
Hub stage env

### PR DESCRIPTION
**Pull request by glenjamin**

### CODE REVIEW

**Comment by romidane** [comment]
@glenjamin how much of this is autogenerated vs hand written?

File: ansible/group_vars/bounce.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ nginx_vhosts:
     state: "present"
     extra_parameters: |
       ignore_invalid_headers off;
-      location / {
+      location /health {
         proxy_pass http://wayland_hub;
```
Comment (jhackettpps) [inline ansible/group_vars/bounce.yml]: This should be `http://wayland_hub/health` I think?

File: ansible/group_vars/bounce.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ nginx_vhosts:
     state: "present"
     extra_parameters: |
       ignore_invalid_headers off;
-      location / {
+      location /health {
         proxy_pass http://berwyn_hub;
```
Comment (jhackettpps) [inline ansible/group_vars/bounce.yml]: As below, `http://berwyn_hub/health`

File: ansible/group_vars/bounce.yml (hunk 1 above)
Comment (glenjamin) [reply ansible/group_vars/bounce.yml]: Hrm, possibly - but it seems to work. I suspect the full incoming request URI gets passed through and it works ok.

File: ansible/roles/hub-containers/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    dest: /usr/local/bin/{{ item }}
+    mode: 0755
+    owner: root
+    group: root
+  with_items:
+    - hub-update
+    - hub-deploy
+    - hub-drush
+  notify:
+    - redeploy containers
```
Comment (jhackettpps) [inline ansible/roles/hub-containers/tasks/main.yml]: Are we now grabbing containers from docker hub or is this purely designed for reloading?

Comment (glenjamin) [reply ansible/roles/hub-containers/tasks/main.yml]: For the staging environment, taking straight from docker hub - yes.

Comment (jhackettpps) [reply ansible/roles/hub-containers/tasks/main.yml]: Aha, OK.

**Pull request merged**